### ISSUE TITLE
Unknown pynest exception

### ISSUE THREAD

**Issue by cristianstoica** [iMicknl/ha-nest-protect#378]
### The problem

Error appeared in log files. Noticed after upgrading from 13.2 to 14.0 but might have been there before.

### What version of this integration (ha-nest-protect) has the issue?

0.4.0b7

### What version of Home Assistant Core has the issue?

2024.12.0

### Device / Model

Nest Protect (Battery) Topaz-2.33 Firmware: 3.5rc3

### Diagnostics information

This error originated from a custom integration.

Logger: custom_components.nest_protect
Source: custom_components/nest_protect/__init__.py:169
integration: Nest Protect (documentation, issues)
First occurred: December 5, 2024 at 1:38:42 PM (13 occurrences)
Last logged: 1:51:29 AM

Unknown pynest exception. Please create an issue on GitHub with your logfile. Updates paused for 1 minute.
Traceback (most recent call last):
  File "/config/custom_components/nest_protect/pynest/client.py", line 317, in subscribe_for_data
    result = await response.json()
             ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/aiohttp_client.py", line 82, in json
    return await super().json(*args, loads=loads, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.13/site-packages/aiohttp/client_reqrep.py", line 1277, in json
    raise ContentTypeError(
    ...<7 lines>...
    )
aiohttp.client_exceptions.ContentTypeError: 200, message='Attempt to decode JSON with unexpected mimetype: text/plain', url='https://czfe06-front01-iad01.transport.home.nest.com/v6/subscribe'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/config/custom_components/nest_protect/__init__.py", line 169, in _async_subscribe_for_data
    result = await entry_data.client.subscribe_for_data(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<4 lines>...
    )
    ^
  File "/config/custom_components/nest_protect/pynest/client.py", line 321, in subscribe_for_data
    raise PynestException(
        f"{response.status} error while subscribing - {result}"
    ) from error
custom_components.nest_protect.pynest.exceptions.PynestException: 200 error while subscribing - 

### Home Assistant log

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks (`)
```
</details>

**Comment by isujosh** [comment]
I've had the exact same issue which has started sometime in the last 10 days or so.

Core: 2024.12.0
Supervisor: 2024.11.4

I'm on integration: 0.4.0b7

**Comment by GSzabados** [comment]
This issue has been logged already, and I have submitted a PR for it and waiting on @iMicknl to review it.

**Comment by iMicknl** [comment]
Duplicate of #345
